examples/knative/powershell/kn-ps-webhook/handler.ps1

```diff
@@ -0,0 +1,123 @@
+Function Process-Init {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Init`n"
+
+   Write-Host "$(Get-Date) - Init Processing Completed`n"
+}
+
+Function Process-Shutdown {
+   [CmdletBinding()]
+   param()
+   Write-Host "$(Get-Date) - Processing Shutdown`n"
+
+   Write-Host "$(Get-Date) - Shutdown Processing Completed`n"
+}
+
+Function Process-Handler {
+   [CmdletBinding()]
+   param(
+      [Parameter(Position=0,Mandatory=$true)]$Headers,
+      [Parameter(Position=1,Mandatory=$true)]$Context
```

Comment:
Technically this is not $Context but $Body and thus signature should be changed accordingly (also in server)